src/cards/card_info.gd

```diff
@@ -1,17 +1,46 @@
-extends Control
+# @tool
+class_name CardInfo
+extends Node
 
+## Docstring
 
+
+# Signals
+
+
+# Enums
+
+
+# Constants
+
+
+# Exported Variables
+
+
+# Variables
+
+
+# Onready Variables
+@onready var background: TextureRect = $Background
 @onready var card_name: Label = %Name
 @onready var icon: TextureRect = %Icon
 @onready var description: Label = %Description
 @onready var rank_icon: TextureRect = %RankIcon
 
+# Built-in Functions
 
+
+# Other Functions
 func initialize(data: CardData) -> void:
```

Comment:
Should we just add `data` as a variable? Storing that reference shouldn't hurt, and could be useful

Reply:
Sure why not